packages/create-contentful-app/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "create-contentful-app",
   "version": "1.1.0-alpha.3",
   "description": "A template for building Contentful Apps",
+  "keywords": ["Contentful", "create-contentful-app", "CCA"],
```

Comment:
@stefanjudis Do you happen to have a list of good keywords we should be using here?

Reply:
I'd drop `CCA`. Primarily `contentful` (I'd go with a lower `c`) is important I'd say. :)